ADAL/resources/mac/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>4.0.6</string>
+	<string>4.0.8</string>
```

Comment:
should it be 4.0.9?
Also, can you update version numbers in other places as well :)?